include/create_shadowentry.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file include/create_shadowentry.php
+ * @brief This script creates posts with UID = 0 for a given public post.
+ *
+ * This script is started from mod/item.php to save some time when doing a post.
+ */
+require_once("boot.php");
+require_once("include/threads.php");
+
+function create_shadowentry_run(&$argv, &$argc) {
```

Comment:
The moment you realize that you are dealing with someone who uses English on a daily basis: When you have to put the text into an online translator to understand some words (like "refrain" or "denote"). ;-)

So you say I shouldn't use the "&" since it doesn't make sense in this context?